### PR TITLE
chore: Fix UI TODOs

### LINKED ISSUE #1955: Fix UI TODOs
Fix all remaining TODOs in UI.

### PR DESCRIPTION
**Pull request by simster7**
Fixes #1955

### CODE REVIEW

**Comment by simster7** [comment]
E2E failure here is fixed in #1964. This branch needs to be rebased after merge.

File: ui/src/app/shared/utils.ts [hunk 1]
```diff
@@ -92,5 +92,33 @@ export const Utils = {
...
+    },
+
+    isWorkflowSuspended(wf: models.Workflow): boolean {
+        if (wf === null || wf.spec === null) {
+            return false;
+        }
+        if (wf.spec.suspend !== undefined && wf.spec.suspend) {
+            return true;
+        }
+        if (wf.status && wf.status.nodes) {
```
Comment (alexec) [inline ui/src/app/shared/utils.ts]: so if any node is suspended - we are suspended?

Comment (simster7) [reply ui/src/app/shared/utils.ts]: Correct

**Comment by simster7** [comment]
Closed in favor of https://github.com/argoproj/argo/pull/1987

**Comment by simster7** [comment]
Closed in favor of https://github.com/argoproj/argo/pull/1989

**Pull request closed without merging**